 04-readline.h-stdio.diff by Gergely Nagy <algernon@debian.org>

 since readline.h uses types defined in stdio.h, that header should
 be included

--- a/libedit/readline/readline.h	2008-04-29 16:53:01.000000000 +1000
+++ b/libedit/readline/readline.h	2009-06-23 08:35:58.000000000 +1000
@@ -31,6 +31,7 @@
 #ifndef _READLINE_H_
 #define _READLINE_H_
 
+#include <stdio.h>
 #include <sys/types.h>
 
 /* list of readline stuff supported by editline library's readline wrapper */
